### PR TITLE
Added enviroment variable to support ingesting merged PRs

### PR DESCRIPTION
**Pull request by oiadebayo**
This pull request introduces modifications focusing on enhanced support for fetching merged PRs.

### Configuration Enhancements:

- Introduced `GET_MERGED_PRS` as a configurable option, allowing for flexibility in whether merged pull requests should be included in data ingestion.

- Used `params={"state": "ALL"}` in the request if `GET_MERGED_PRS` is set to `True`, enabling the ingestion of all pull request states.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 PORT_API_URL = config("PORT_API_URL", default="https://api.getport.io/v1")
 WEBHOOK_SECRET = config("WEBHOOK_SECRET", default="bitbucket_webhook_secret")
 IS_VERSION_8_7_OR_OLDER = config("IS_VERSION_8_7_OR_OLDER", default=False)
+GET_MERGED_PRS = config("GET_MERGED_PRS", default=False)
```
Comment (PeyGis) [inline app.py]: When you pass ALL, it brings OPEN, MERGED and DECLINED prs. Based on that, can we rename the flag to  INGEST_ALL_PR_STATES or something equivalent?

Alternatively, you could make it PULL_REQUEST_STATE and allow them to pass ALL, OPEN, MERGED or DECLINED. then you set the default to OPEN.

**Review by PeyGis — CHANGES_REQUESTED** [review]
just a minor comment then we're good to go

File: app.py (hunk 1 above)
Comment (oiadebayo) [reply app.py]: OK

**Review by PeyGis — APPROVED** [review]
Approved

**Pull request merged**